examples/zkdetect/main.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
Can't we use `testing` examples instead? http://golang.org/pkg/testing/#hdr-Examples